broadcast-monitoring/infrastructure/elemental/custom_resources/custom-resource-py/requirements.txt

```diff
@@ -0,0 +1 @@
+crhelper
```

Comment:
Might be useful to include version of **crhelper**

Reply:
updated